contracts/EverlongStrategy.sol

```diff
@@ -502,8 +502,9 @@ contract EverlongStrategy is BaseStrategy {
             // Hyperdrive's minimum transaction amount.
             if (
                 totalPositionValue >
-                (_targetOutput - output + _poolConfig.minimumTransactionAmount)
-                    .mulUp(ONE + partialPositionClosureBuffer)
+                (_targetOutput - output + _minimumTransactionAmount()).mulUp(
```

Comment:
This looks right to me. I scrutinized this because I wanted to be sure that we weren't potentially comparing a bond amount to shares. This looks right since it's `shares > share`, but I do want to point out that if the LHS was in bonds, we would want to use `_poolConfig.minimumTransactionAmount` rather than `_minimumTransactionAmount`. Seems like this is being done correctly in the PR though, so this is just a note.